tests/ui/span/issue-34264.stderr

```diff
@@ -1,8 +1,8 @@
-error: expected one of `:`, `@`, or `|`, found `<`
+error: expected one of `!`, `(`, `...`, `..=`, `..`, `::`, `:`, `{`, or `|`, found `<`
```

Comment:
If we can, I'd like to revert whatever causes this message change, as I feel the previous one did a good job leading people towards the problem of a missing `:`

Reply:
This is reverted now! ;)